docker/manage

```diff
@@ -662,6 +671,12 @@ case "${COMMAND}" in
     docker-compose up --scale vcr-worker=2 -d ${_startupParams} 
     docker-compose logs -f
     ;;
+  scale|upscale)
+    _startupParams=$(getStartupParams --force-recreate $@)
+    configureEnvironment "$@"
```

Comment:
Can we add a prompt here that asks the user to comment the exposed ports for the agent and expects input before proceeding, to avoid confusion?

Reply:
A warning with a press any key to continue, would be helpful.

Reply:
Done